packages/kas/package.json

```diff
@@ -1,38 +1,39 @@
 {
-  "publishConfig": {
-    "access": "public"
-  },
-  "name": "@khanacademy/kas",
-  "version": "0.2.3",
-  "description": "A lightweight JavaScript CAS for comparing expressions and equations.",
-  "module": "dist/es/index.js",
-  "main": "dist/index.js",
-  "source": "src/index.js",
-  "scripts": {
-    "gen:parsers": "node src/parser-generator.js",
-    "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Khan/KAS.git"
-  },
-  "keywords": [
-    "parsing",
-    "equation",
-    "expression",
-    "algebra",
-    "symbolic"
-  ],
-  "author": "alopatin",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/Khan/KAS/issues"
-  },
-  "devDependencies": {
-    "jison": "0.4.15",
-    "underscore": "1.4.4"
-  },
-  "peerDependencies": {
-    "underscore": "1.4.4"
-  }
-}
\ No newline at end of file
+    "name": "@khanacademy/kas",
+    "description": "A lightweight JavaScript CAS for comparing expressions and equations.",
+    "author": "alopatin",
+    "license": "MIT",
+    "version": "0.2.3",
+    "publishConfig": {
+        "access": "public"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Khan/kas.git"
+    },
+    "bugs": {
+        "url": "https://github.com/Khan/perseus/issues"
+    },
+    "module": "dist/es/index.js",
+    "main": "dist/index.js",
+    "source": "src/index.js",
+    "scripts": {
+        "gen:parsers": "node src/parser-generator.js",
+        "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
+    },
+    "dependencies": {},
```

Comment:
I'm opting to put empty top-level keys in these files as hints to where they should be if someone needs to add a new dev, for example (and if these are here, `yarn` will keep `dependencies` here instead of somewhere else.